addons/point_of_sale/tools/posbox/configuration/odoo.conf

```diff
@@ -7,3 +7,4 @@ pidfile = /var/run/odoo/odoo.pid
 limit_time_cpu = 600
 limit_time_real = 1200
 max_cron_threads = 0
+server_wide_modules=hw_drivers,hw_escpos,hw_posbox_homepage,point_of_sale,web
```

Comment:
technically, `web` is always loaded, so we can remove it see
https://github.com/odoo/odoo/blob/14e28aff5ed45c35d9f889efff92b27410714029/odoo/service/server.py#L1259

But maybe we should keep it for clarity sake (as it does not change anything afterall)

Reply:
interesting info, but I still think we should keep it
if one day someone who doesn't know that "web" is always loaded will work with the IoT it will be easier for him if he sees it here